### DISCUSSION TITLE
6 Months of Video

### DISCUSSION THREAD

**Discussion by abiyarb** [talmolab/sleap#1812] [Help!]
Hi All!

What would be the best way to train and use a model to predict 6 months of video? The video is split up into hour-long videos that are sped up. The location does not change and the identities of the three targets do not change either. 

I have considered tracking each day as an individual project and just using the same trained model to predict on them, but I do not know how effective this will be. 

Any help is appreciated!

**Comment by talmo** [comment]
Hi @abiyarb,

Apologies for the delay!

That's a lot of data!

In terms of training the model, just be sure that you have representative examples from across the dataset.

If you have a model that works well, then I'd recommend using the [`sleap-track` CLI](https://sleap.ai/guides/cli.html#sleap-track) in a script where you call it in a loop (or parallelize across video files if you have access to multiple GPUs/machines).

Let us know if you have any questions!

Cheers,

Talmo

**Comment by AmanRathoreP** [comment]
I guess you should try [pytorch version of sleap](https://github.com/talmolab/sleap-nn) for training as it allows you to train on multiple GPUs in parallel with pretty easy setyp
